src/main/features/response/views/eligibility/send-your-response-by-email.njk

```diff
@@ -0,0 +1,98 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+{% from "form.njk" import csrfProtection, submitButton %}
+
+{% set title = t('Money claim') %}
+{% set heading = t('Send your response by email') %}
+
+{% macro newWindow() %}<span class="font-xsmall">{{ t('(opens in a new window)') }}</span>{% endmacro %}
```

Comment:
please extract to common frontend.
update all current places it is used in a follow up PR
REQ

Reply:
👍

What if we had `externalLink` macro that would do:

```
{% macro externalLink(href, text) %}
  <a href="{{ href }}" target="_blank">{{ text }}</a>
  <span class="font-xsmall">{{ t('(opens in a new window)') }}</span>
{% endmacro %}
```

I dislike the `newWindow` name.